.github/workflows/release-deb.yml

```
@@ -29,24 +20,19 @@ jobs:
+      - name: Build package
         run: DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -us -uc -b
-
-      - name: Upload .deb to release
-        run: |
-          DEB=$(ls pastebinit_*.deb | head -1)
-          gh release upload "${{ inputs.tag }}" "$DEB"
+      - name: Upload package
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ github.token }}
+        run: gh release upload "${{ inputs.tag }}" "$(ls pastebinit_*.deb | head -1)"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:1:42: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)